pkg/scaffold/v2/authproxyservice.go

```diff
@@ -14,52 +14,46 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package resource
+package v2
 
 import (
-	"fmt"
 	"path/filepath"
 
 	"sigs.k8s.io/kubebuilder/pkg/scaffold/input"
 )
 
-var _ input.File = &Role{}
+var _ input.File = &AuthProxyService{}
 
-// Role scaffolds the config/manager/group_role_rbac.yaml file
-type Role struct {
+// AuthProxyService scaffolds the config/rbac/auth_proxy_role.yaml file
```

Comment:
This comment isn't correct. `rbac/auth_proxy_role` ??

Reply:
Done